contrib/babelfishpg_tsql/src/multidb.c

```diff
@@ -1334,18 +1334,30 @@ get_physical_user_name(char *db_name, char *user_name, bool suppress_error)
 const char *
```

Comment:
Lets make return type as `char *` from `const char *` for all such APIs where we returning palloc'd copy.

Reply:
Made the return type as `char *` from `const char * `